src/internal/testpachd/mock_transaction.go

```diff
@@ -31,6 +31,7 @@ func (mock *mockCreatePipelineInTransaction) Use(cb createPipelineInTransactionF
 }
 
 type ppsTransactionAPI struct {
+	ppsServerAPI
```

Comment:
The PPS server we give to the serviceenv has to expose all the API methods and the transaction methods, this seemed like the simplest way to achieve that (and the PFS tests pass) but I don't know this part of the codebase very well.